src/components/utils/index.js

```diff
@@ -52,31 +52,15 @@ export const getUnsupportedMobileBrowserError = () => {
 // Copied from https://github.com/muaz-khan/DetectRTC/blob/master/DetectRTC.js
 export const isDesktop = !(/Android|webOS|BB10|BlackBerry|IEMobile|Opera Mini|Mobile|mobile/i.test(navigator.userAgent || '')) && !isIOS
 
+const isWindows = navigator.userAgent.indexOf('Windows') > -1;
```

Comment:
Minor: Can also be written as `!navigator.userAgent.includes('Windows')` which I think is more readable.

Reply:
Yes, I didn't remember if we had polyfill for IE11 in this part of the code or not. Will change it